mava/configs/network/rnn.yaml

```diff
@@ -27,3 +27,15 @@ critic_network:
     layer_sizes: [128]
     use_layer_norm: False
     activation: relu
+
+q_network:
```

Comment:
We can, but by the same logic we can use the same one for actor and critic. Also, I am not sure how the network hyperparameters will look for qmix just yet, so it may be good to start off being explicit about where which variables are used! Not a hill I'm willing to die on, but my preference is to keep this.